docs/usage/shoot_high_availability_best_practices.md

```diff
@@ -248,7 +245,7 @@ spec:
       ...
 ```
 
-And please do not specify a PDB of `maxUnavailable` being 0 or similar. That's pointless, even detrimental, as it blocks then even useful operations, forces always the hard timeouts that are less graceful and it doesn't make sense in the context of HA. You cannot "force" HA by preventing voluntary pod disruptions, you must work with the pod disruptions in a resilient way. Besides, PDBs are really only about voluntary pod disruptions - something bad can happen to a node/pod at any time and PDBs won't make this reality go away for you.
+And please do not specify a PDB of `maxUnavailable` being 0 or similar. That's pointless, even detrimental, as it blocks then even useful operations, forces always the hard timeouts that are, well... less graceful and it doesn't make sense in the context of HA. You cannot "force" HA by preventing voluntary pod disruptions, you must work with the pod disruptions in a resilient way. Besides, PDBs are really only about voluntary pod disruptions - something bad can happen to a node/pod at any time and PDBs won't make this reality go away for you.
```

Comment:
I prefer the previous variant.

```suggestion
And please do not specify a PDB of `maxUnavailable` being 0 or similar. That's pointless, even detrimental, as it blocks then even useful operations, forces always the hard timeouts that are less graceful and it doesn't make sense in the context of HA. You cannot "force" HA by preventing voluntary pod disruptions, you must work with the pod disruptions in a resilient way. Besides, PDBs are really only about voluntary pod disruptions - something bad can happen to a node/pod at any time and PDBs won't make this reality go away for you.
```

Reply:
Yeah, I found it more entertaining this way. I will take your your advice and remove it again.